src/lib/Preview.js

```diff
@@ -488,31 +484,40 @@ class Preview extends EventEmitter {
      * @return {void}
      */
     download() {
-        const { apiHost, queryParams } = this.options;
-
+        const downloadErrorMsg = __('notification_cannot_download');
         if (!canDownload(this.file, this.options)) {
+            this.ui.showNotification(downloadErrorMsg);
```

Comment:
Why would we show a button if they can't download? If we don't know till reps come back this will get interesting...

Reply:
This is a public method so one could call `preview.download()` directly

Reply:
sounds good!